builtin/commit-graph.c

```diff
@@ -179,6 +190,8 @@ static int graph_write(int argc, const char **argv)
 		OPT_END(),
 	};
 
+	/* Turn on progress by default for terminals, off for stderr */
```

Comment:
- I'm not sure this comment is necessary.  `pack-object`, `prune`, `fsck`, etc. have a similar call to `isatty(2)` with no comment
- If we leave the comment, the "off for stderr" might be a bit misleading.  The `isatty(2)` call is to check if `stderr` is being written to a terminal and so the default is to turn on progress when `stderr` refers to a terminal.

cc: @derrickstolee @jeffhostetler for advice on adding comments

Reply:
I thnk you are right... I don't think anyone will care about that comment other than me. I am removing it.